api/test/wfl/tools/workloads.clj

```diff
@@ -66,6 +67,17 @@
    :project  (format "(Test) %s" @git-branch)
    :items    [{:src src :dst dst}]})
 
+(defn xx-workload-request
+  [identifier]
+  "A whole genome sequencing workload used for testing."
+  (let [test-storage "gs://broad-gotc-test-storage/exome/plumbing/truth/master/"]
```

Comment:
Can you add the test inputs to `gs://broad-gotc-dev-wfl-ptc-test-inputs/` for consistency with the other tests?

Reply:
These aren't my test inputs - they're from trevyn.